seedsource_core/django/seedsource/tasks/constraints.py

```diff
@@ -320,3 +321,12 @@ def get_mask(self, lat, lon, distance):
             dtype=numpy.uint8
         )
 
+class GeometryConstraint(Constraint): 
```

Comment:
@kennino This needs and extra line separating it from the previous class definitions: https://www.python.org/dev/peps/pep-0008/#blank-lines

> Surround top-level function and class definitions with two blank lines.